lib/model/model.go

```diff
@@ -876,6 +876,9 @@ func (m *model) Completion(device protocol.DeviceID, folder string) (FolderCompl
 	// We want completion for all (shared) folders as an aggregate.
 	var comp FolderCompletion
 	for _, fcfg := range m.cfg.FolderList() {
+		if fcfg.Paused {
```

Comment:
There's a race condition here in that the folder might not be paused when we look now, but it might be when we calculate the completion. I suggest an `errors.Is(err, ErrFolderPaused)` check on the returned error in the loop instead